tests/examples/step-12.diff

```diff
@@ -0,0 +1,16 @@
+20a21
+> #include "../example_test.h"
+495c496,500
+<     solver.solve(system_matrix, solution, right_hand_side, preconditioner);
+---
+>     check_solver_within_range(std::cout,
+> 			      solver.solve(system_matrix, solution, right_hand_side, preconditioner),
+> 			      solver_control.last_step(),
+> 			      3,
+> 			      100);
```

Comment:
It seems this isn't conforming with `clang-format` (which might not matter much).

Reply:
Right, it doesn't matter because nobody is every going to get to see the result. Let's not worry about it.